package.json

```diff
@@ -3,7 +3,7 @@
   "displayName": "Calva: Clojure & ClojureScript Interactive Programming",
   "description": "Integrated REPL, formatter, Paredit, and more. Powered by cider-nrepl and clojure-lsp.",
   "icon": "assets/calva.png",
-  "version": "2.0.485",
+  "version": "2.0.486",
```

Comment:
I think you may have based your changes off of `dev` instead of `published`. 486 is the next version.

Reply:
I changed base